doc/nrf/ug_thingy53.rst

```diff
@@ -100,9 +100,11 @@ Building and programming using Visual Studio Code
 
 Complete the following steps after installing the `nRF Connect for Visual Studio Code`_ extension in Microsoft Visual Studio Code:
 
-.. include:: /includes/vsc_build_and_run_nrf53.txt
+.. |sample_path_vsc| replace:: :file:`nrf/samples/bluetooth/peripheral_lbs`
 
-6. Select the ``thingy53_nrf5340_cpuapp`` as the target board.
+.. include:: /includes/vsc_build_and_run.txt
+
+8. Select the ``thingy53_nrf5340_cpuapp`` as the target board.
```

Comment:
Can we cover the 'Build configuration' steps also in the "vsc_build_and_run.txt" - We could just replace the target board options.
This way, in future,  for any new changes in the VSC documentation, we could just edit the "vsc_build_and_run.txt" .

Reply:
Sure, let me work on this.

Reply:
Done.